_includes/v2.1/zone-configs/create-a-replication-zone-for-a-system-range.md

```diff
@@ -0,0 +1,37 @@
+In addition to the databases and tables that are visible via the SQL interface, CockroachDB stores internal data in what are called system ranges. CockroachDB comes with pre-configured replication zones for some of these ranges:
+
+Zone Name | Description
+----------|-----------------|------------
+`.meta` | The "meta" ranges contain the authoritative information about the location of all data in the cluster.<br><br>Because historical queries are never run on meta ranges and it is advantageous to keep these ranges smaller for reliable performance, CockroachDB comes with a **pre-configured** `.meta` replication zone giving these ranges a lower-than-default `gc.ttlseconds`.<br><br>If your cluster is running in multiple datacenters, it's a best practice to configure the meta ranges to have a copy in each datacenter.
+`.liveness` | The "liveness" range contains the authoritative information about which nodes are live at any given time.<br><br>Just as for "meta" ranges, historical queries are never run on the liveness range, so CockroachDB comes with a **pre-configured** `.liveness` replication zone giving this range a lower-than-default `gc.ttlseconds`.<br><br>If this range is unavailable, the entire cluster will be unavailable, so giving it a high replication factor is strongly recommended.
+`.timeseries` | The "timeseries" ranges contain monitoring data about the cluster that powers the graphs in CockroachDB's Admin UI. If necessary, you can add a `.timeseries` replication zone to control the replication of this data.
+`.system` | There are system ranges for a variety of other important internal data, including information needed to allocate new table IDs and track the status of a cluster's nodes. If necessary, you can add a `.system` replication zone to control the replication of this data.
+
+To control replication for one of the above sets of system ranges, use the `CONFIGURE ZONE` statement to define the values you want to change (other values will not be affected):
+
+{% include copy-clipboard.html %}
+~~~ sql
+> ALTER RANGE meta CONFIGURE ZONE USING num_replicas = 7, gc.ttlseconds = 100000;
```

Comment:
Given the description above, does this mean the cluster is running in 7 data centers?  What does this GC TTL mean in the context of the info earlier on the page?  Is this new GC TTL setting here just to show the syntax? (It appears to be about 10% bigger than default, which I guess is why I ask).